test/ngResource/resourceSpec.js

```diff
@@ -338,6 +338,17 @@ describe("resource", function() {
   });
 
 
+  it('should encode & in query params unless in query param value', function() {
```

Comment:
I would say close but not quite.
I would be happy to have them merged into one `it` block (keeping all 3 cases).

Feel free to merge them.